pom.xml

```diff
@@ -1723,6 +1726,13 @@
       </dependency>
 
       <dependency>
+        <groupId>com.ahome-it</groupId>
+        <artifactId>lienzo-tests</artifactId>
+        <version>${version.com.ahome-it.lienzo-tests}</version>
+        <scope>test</scope>
```

Comment:
Nit picking, but we should not define scope in dependencyManagement. Modules depending on this one are in charge of deciding which scope they need. This would work for most of the case (or maybe all of them), but makes it slightly harder to understand since it changes the default scope from `compile` to `test` (e.g. if the module depending on this artifact does not specify scope, it would be `test` instead of the otherwise defaulted `compile`)